examples/fortran/SHCilmPlus/meson.build

```diff
@@ -0,0 +1,11 @@
+TestCilmPlus = executable(
+  'TestCilmPlus',
+  'TestCilmPlus.f95',
+  link_with: shtools)
+
+test(
+  'CilmPlus',
+  TestCilmPlus,
+  args: [
+    meson.current_source_dir() / '../../ExampleDataFiles',
+  ])
```

Comment:
This test timed out on my laptop. I would suggest adding this argument to `test`:
```
timeout: 60
```